src/components/Select/Select.stories.tsx

```diff
@@ -82,6 +83,11 @@ export const WithControls: Story = {
     },
     jest: ["Select.test.tsx"],
   },
+  play: async ({ canvasElement }) => {
+    const screen = within(canvasElement);
+    const select = screen.getByRole("combobox");
+    await userEvent.selectOptions(select, "green");
```

Comment:
Add a few more events to select other values. Marty and I still go back and forth on this, but add one expectation that the select's value is what you expect it to be.